plugin/src/main/java/net/pl3x/map/configuration/Advanced.java

```diff
@@ -748,75 +753,76 @@ Setting a color to black (#000000) will make it invisible.""")
         put(Blocks.POLISHED_BLACKSTONE_SLAB, 0x353139);
         put(Blocks.POLISHED_BLACKSTONE_STAIRS, 0x353139);
         put(Blocks.POLISHED_BLACKSTONE_WALL, 0x353139);
-        put(Blocks.POLISHED_DEEPSLATE, 0x484849);
+        put(Blocks.POLISHED_BLACKSTONE, 0x353139);
         put(Blocks.POLISHED_DEEPSLATE_SLAB, 0x484849);
         put(Blocks.POLISHED_DEEPSLATE_STAIRS, 0x484849);
         put(Blocks.POLISHED_DEEPSLATE_WALL, 0x484849);
-        put(Blocks.POLISHED_DIORITE, 0xC3C3C5);
+        put(Blocks.POLISHED_DEEPSLATE, 0x484849);
         put(Blocks.POLISHED_DIORITE_SLAB, 0xC3C3C5);
         put(Blocks.POLISHED_DIORITE_STAIRS, 0xC3C3C5);
-        put(Blocks.POLISHED_GRANITE, 0x9B6B59);
+        put(Blocks.POLISHED_DIORITE, 0xC3C3C5);
         put(Blocks.POLISHED_GRANITE_SLAB, 0x9B6B59);
         put(Blocks.POLISHED_GRANITE_STAIRS, 0x9B6B59);
+        put(Blocks.POLISHED_GRANITE, 0x9B6B59);
         put(Blocks.POPPY, 0x8A3F26);
         put(Blocks.POTATOES, 0x46872A);
-        put(Blocks.POTTED_ACACIA_SAPLING, 0x777618);
-        put(Blocks.POTTED_ALLIUM, 0xA089B9);
-        put(Blocks.POTTED_AZALEA, 0x657D30);
-        put(Blocks.POTTED_AZURE_BLUET, 0xACCE82);
-        put(Blocks.POTTED_BAMBOO, 0x5D9013);
-        put(Blocks.POTTED_BIRCH_SAPLING, 0x81A251);
-        put(Blocks.POTTED_BLUE_ORCHID, 0x30A3AA);
-        put(Blocks.POTTED_BROWN_MUSHROOM, 0x9A755C);
-        put(Blocks.POTTED_CACTUS, 0x58822D);
-        put(Blocks.POTTED_CORNFLOWER, 0x507896);
-        put(Blocks.POTTED_CRIMSON_FUNGUS, 0x8F2C1D);
-        put(Blocks.POTTED_CRIMSON_ROOTS, 0x7F082A);
-        put(Blocks.POTTED_DANDELION, 0x9EB02E);
-        put(Blocks.POTTED_DARK_OAK_SAPLING, 0x3C5B1E);
-        put(Blocks.POTTED_DEAD_BUSH, 0x6D5029);
-        put(Blocks.POTTED_FERN, 0x007C00);
-        put(Blocks.POTTED_FLOWERING_AZALEA, 0x9D5CAB);
-        put(Blocks.POTTED_JUNGLE_SAPLING, 0x305111);
-        put(Blocks.POTTED_LILY_OF_THE_VALLEY, 0x7DB061);
-        put(Blocks.POTTED_MANGROVE_PROPAGULE, 0x5FAF54);
-        put(Blocks.POTTED_OAK_SAPLING, 0x4E6A29);
-        put(Blocks.POTTED_ORANGE_TULIP, 0x5E8E1F);
-        put(Blocks.POTTED_OXEYE_DAISY, 0xB7CC91);
-        put(Blocks.POTTED_PINK_TULIP, 0x649E50);
-        put(Blocks.POTTED_POPPY, 0x8A3F26);
-        put(Blocks.POTTED_RED_MUSHROOM, 0xDA4740);
-        put(Blocks.POTTED_RED_TULIP, 0x5B8122);
-        put(Blocks.POTTED_SPRUCE_SAPLING, 0x2D3D25);
-        put(Blocks.POTTED_WARPED_FUNGUS, 0x4C7059);
-        put(Blocks.POTTED_WARPED_ROOTS, 0x148B7D);
-        put(Blocks.POTTED_WHITE_TULIP, 0x5FA548);
-        put(Blocks.POTTED_WITHER_ROSE, 0x292C17);
-        put(Blocks.POWDER_SNOW, 0xF8FDFD);
+        put(Blocks.POTTED_ACACIA_SAPLING, 0x000000);
```

Comment:
I definitely want all of these colored. I had them set to be the same as their unpotted counterparts.